### DISCUSSION TITLE
Must use import to load ES Module

### DISCUSSION THREAD

**Discussion by lausoffici** [nolimits4web/swiper#5009] [Q&A] [answered]
Im trying to build my next.js app and I don't understand why I'm getting this error:

![image](https://user-images.githubusercontent.com/31258347/134972908-95a84053-34d1-408e-8c9a-9bb5f2df971e.png)

I use ES6 modules. I tried adding "type": "module" in package.json.

Here is the code:
![image](https://user-images.githubusercontent.com/31258347/134973155-7f45bbde-7b52-4500-ab06-3b0dfea2ea68.png)

Version of Swiper: 7.0.6

I hope someone knows about this problem.

**Comment by nolimits4web** [comment, marked as answer]
Make sure you are using latest version of Next.js and enable experimental modules in config. For reference check Swiper's website which also uses Next.js https://github.com/nolimits4web/swiper-website

**Reply by lausoffici** [reply]
Thank you! I solve my build errors. I didn't know that config, so i will take a look! Thank you
